src/Sylius/Bundle/CurrencyBundle/spec/Form/Type/CurrencyTypeSpec.php

```diff
@@ -39,12 +39,12 @@ function it_is_a_form_type()
     function it_should_build_form_with_proper_fields(FormBuilder $builder)
     {
         $builder
-            ->add('code', 'currency', Argument::any())
+            ->add('exchangeRate', 'number', Argument::any())
             ->willReturn($builder)
         ;
 
         $builder
-            ->add('exchangeRate', 'number', Argument::any())
+            ->addEventSubscriber(Argument::type('Sylius\Bundle\ResourceBundle\Form\EventSubscriber\AddCodeFormSubscriber'))
```

Comment:
I think @pamil meant to add a use statment and use ``::class`` instead of string. :)